### PR TITLE
Show tflite version when interpreter initialize

### PR DESCRIPTION
**Pull request by joyalbin**

### CODE REVIEW

File: tensorflow/lite/interpreter.cc [hunk 1]
```diff
@@ -60,8 +61,9 @@ Interpreter::Interpreter(ErrorReporter* error_reporter)
   // Only log initialization once per-process to avoid log spam.
   static std::once_flag init_log_once_flag;
   std::call_once(init_log_once_flag, []() {
-    // TODO(b/128420794): Include the TFLite runtime version in the log.
-    TFLITE_LOG_PROD(TFLITE_LOG_INFO, "Initialized TensorFlow Lite runtime.");
+    TFLITE_LOG_PROD(TFLITE_LOG_INFO,
+                    "Initialized TensorFlow Lite runtime version %d.",
+                    TFLITE_SCHEMA_VERSION);
```
Comment (haozha111) [inline tensorflow/lite/interpreter.cc]: I thought TFLITE_SCHEMA_VERSION does not encode the TF Lite runtime version (e.g. v1.1.3):
https://github.com/tensorflow/tensorflow/blob/fa5e3a968215688446c395b90f5313686db29df3/tensorflow/lite/version.h#L21

And I'm not sure where to get that information(seems not possible).

Comment (joyalbin) [reply tensorflow/lite/interpreter.cc]: @haozha111 I have taken reference from the application source code ./tensorflow/lite/java/src/main/native/tensorflow_lite_jni.cc

Could you please make me clear if there something has to be changed?

Comment (haozha111) [reply tensorflow/lite/interpreter.cc]: Currently TF Lite doesn't provide an API to expose the runtime version. However we are currently working on a CL to implement that. Should be checked-in in a few days. Later on let's leverage that change in your PR.

Thanks a lot!

Comment (haozha111) [reply tensorflow/lite/interpreter.cc]: I think we have added a new API to get TF Lite runtime version:
https://github.com/tensorflow/tensorflow/blob/1056e30399af74b3a9d84145cb184eefaa121ef7/tensorflow/lite/experimental/c/c_api.h#L65

Could you take another look? Thanks.

**Comment by joyalbin** [comment]
@haozha111 I have used `TFLITE_VERSION_STRING` directly because `tensorflow/lite/experimental` is not linked as of now. Could you please review?

**Comment by joyalbin** [comment]
@haozha111 Could you please kindly review this PR, I have resolved the conflict and uploaded

**Review by haozha111 — APPROVED** [review]

**Comment by jdduke** [comment]
Please don't land this yet, we have some internal tests that it will break. I already have a CL to update the version, so I'm going to close this for now.

**Pull request closed without merging**